### PR TITLE
Added several levels of brightness when click button 1

### PR DESCRIPTION
**Pull request by alcar21**
I added several brightness levels to partially cover this issue.

https://github.com/BitMaker-hub/NerdMiner_v2/issues/91

### CODE REVIEW

File: src/NerdMinerV2.ino.cpp [hunk 1]
```diff
@@ -43,14 +43,15 @@ void alternate_screen_state() {
   // digitalWrite(TFT_BL, !screen_state);
   if (brightness == 255) {
     brightness = 0;
-  } else if (brightness == 250) {
+  } else if (brightness == 200) {
     brightness = 255;
   } else {
     brightness += 50;
```
Comment (golden-guy) [inline src/NerdMinerV2.ino.cpp:49]: Using 51 as interval would save you an else if

File: src/NerdMinerV2.ino.cpp [hunk 2]
```diff
@@ -32,14 +32,25 @@ TFT_eSprite background = TFT_eSprite(&tft);  // Invoke library sprite
 
 unsigned long start = millis();
 const char* ntpServer = "pool.ntp.org";
+byte brightness = 255;
```
Comment (golden-guy) [inline src/NerdMinerV2.ino.cpp:35]: I'd suggest to use unsigned int, if you anyway need to cast it.

File: src/NerdMinerV2.ino.cpp (hunk 1 above)
Comment (alcar21) [reply src/NerdMinerV2.ino.cpp:49]: yep, I will change it! It's good idea!

File: src/NerdMinerV2.ino.cpp (hunk 2 above)
Comment (alcar21) [reply src/NerdMinerV2.ino.cpp:35]: But the byte is smaller and the max size of brightness is 255, in memory is more efficient.

**Pull request closed without merging**